dali/benchmark/operator_bench.h

```diff
@@ -35,26 +35,26 @@ class OperatorBench : public DALIBenchmark {
 
     auto op_ptr = InstantiateOperator(op_spec);
 
-    std::vector<std::shared_ptr<Tensor<CPUBackend>>> data_in(batch_size);
-    std::vector<std::shared_ptr<Tensor<CPUBackend>>> data_out(batch_size);
-    for (auto &in_ptr : data_in) {
+    auto data_in = std::make_shared<TensorVector<CPUBackend>>(batch_size);
```

Comment:
Why not unique_ptr?

Reply:
Or allocate on the stack?

Reply:
Beacuse it's benchmark and I don't really care if the setup code does one heap allocation or uses something from stack. Also, it is the type used by the workspace.